apps/meteor/server/services/messages/hooks/BeforeSaveCheckMAC.ts

```diff
@@ -0,0 +1,24 @@
+import { MeteorError, Omnichannel } from '@rocket.chat/core-services';
+import type { IRoom } from '@rocket.chat/core-typings';
+import { isOmnichannelRoom, type IMessage } from '@rocket.chat/core-typings';
+
+export class BeforeSaveCheckMAC {
+	async isWithinLimits({ message, room }: { message: IMessage; room: IRoom }): Promise<void> {
```

Comment:
should we keep the "skip edited messages" if? 👀

Reply:
![image](https://github.com/RocketChat/Rocket.Chat/assets/8591547/e47a014b-483b-46e4-8271-816d367e4071)

Reply:
it is there actually:

https://github.com/RocketChat/Rocket.Chat/pull/31231/files#diff-6d79e082eda977d9389a7d87f152ba71fc3566c53696d7be838b4893e33af933R139

Reply:
![image](https://github.com/RocketChat/Rocket.Chat/assets/11577696/af8e2b16-df20-4e46-bf20-ab8e43861410)